src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -694,12 +688,10 @@ MulticopterPositionControl::Run()
 				limit_thrust_during_landing(local_pos_sp);
 			}
 
-			// Fill attitude setpoint. Attitude is computed from yaw and thrust setpoint.
 			vehicle_attitude_setpoint_s attitude_setpoint{};
-			attitude_setpoint = ControlMath::thrustToAttitude(matrix::Vector3f(local_pos_sp.thrust), local_pos_sp.yaw);
-			attitude_setpoint.yaw_sp_move_rate = _control.getYawspeedSetpoint();
-			attitude_setpoint.fw_control_yaw = false;
-			attitude_setpoint.apply_flaps = false;
+			attitude_setpoint.timestamp = hrt_absolute_time();
```

Comment:
could you reuse the ` hrt_absolute_time();` of above (`local_pos_sp.timestamp = hrt_absolute_time();`) ?